addons/account/views/account_reconcile_model_views.xml

```diff
@@ -102,65 +102,41 @@
+                                                <label for="match_amount" attrs="{'invisible': [('rule_type', '=', 'writeoff_button')]}"/>
+                                                <div class="d-flex gap-2" attrs="{'invisible': [('rule_type', '=', 'writeoff_button')]}">
+                                                    <field name="match_amount" style="width:60%"/>
                                                     <field name="match_amount_min"
-                                                           style="width: 30% !important"
-                                                           class="oe_inline"
                                                            attrs="{'invisible': [('match_amount', 'in', (False, 'lower'))], 'required': [('match_amount', '!=', False)]}"/>
                                                     <span class="o_form_label"
                                                           attrs="{'invisible': [('match_amount', '!=', 'between')]}">and</span>
                                                     <field name="match_amount_max"
-                                                           style="width: 30% !important"
-                                                           class="oe_inline"
                                                            attrs="{'invisible': [('match_amount', 'in', (False, 'greater'))], 'required': [('match_amount', '=', 'between')]}"/>
                                                 </div>
                                                 <label for="allow_payment_tolerance"
-                                                       attrs="{'invisible': [('rule_type', '!=', 'invoice_matching')]}"/>
-                                                <div attrs="{'invisible': [('rule_type', '!=', 'invoice_matching')]}">
-                                                    <field name="allow_payment_tolerance" class="oe_inline"/>
-                                                    <span attrs="{'invisible': [('allow_payment_tolerance', '=', False)]}">
-                                                        <field name="payment_tolerance_param"
-                                                               style="width: 30% !important"
-                                                               class="oe_inline"/>
-                                                        <span class="o_form_label"
-                                                              style="width: 3% !important">   </span> <!-- Make a little space. -->
-                                                        <field name="payment_tolerance_type"
-                                                               style="width: 60% !important"
-                                                               class="oe_inline"/>
+                                                       attrs="{'invisible': [('rule_type', '!=', 'invoice_matching'), ('rule_type', '=', 'writeoff_button')]}"/>
```

Comment:
```suggestion
                                                       attrs="{'invisible': [('rule_type', '=', 'writeoff_button')]}"/>
```

Reply:
This is not part of the commit message (or is it the alignment issue thing?), and I'm not sure why we would support `writeoff_suggestion` now for the payment tolerance?

Reply:
I removed the <div attrs="{'invisible': [('rule_type', '!=', 'invoice_matching')]}"> for the alignement issues, so i put the invisible on all field present in that div

Reply:
Sorry. My suggestion was wrong. It should be:
```suggestion
                                                       attrs="{'invisible': [('rule_type', '!=', 'invoice_matching')]}"/>
```